gramps/gen/config.py

```diff
@@ -172,7 +172,7 @@ def emit(key):
 register("behavior.startup", 0)
 register("behavior.surname-guessing", 0)
 register("behavior.translator-needed", True)
-register("behavior.use-tips", False)
+register("behavior.use-tips", True)
 register("behavior.welcome", 100)
 register("behavior.web-search-url", "http://google.com/#&q=%(text)s")
```

Comment:
While you are at it consider changing  the "http://" to the secure version "https://"  please?  for this and any other url you can find.

Reply:
A search for `http://www` and replace with `https://www`
would get most of the offenders without touching the parsers'

```
if path.startswith(("http://", "https://"))
```

Reply:
I've created #1979 to do this.
Alas, it is not as simple as search and replace http with https.